app/assets/javascripts/app/views/help_view.js

```diff
@@ -51,19 +51,22 @@ app.views.Help = app.views.StaticContentView.extend({
       title_sharing: Diaspora.I18n.t( 'sharing.title' ),
       title_tags: Diaspora.I18n.t( 'tags.title' ),
       title_keyboard_shortcuts: Diaspora.I18n.t( 'keyboard_shortcuts.title' ),
-      title_miscellaneous: Diaspora.I18n.t( 'miscellaneous.title' ),
-    }
+      title_miscellaneous: Diaspora.I18n.t( 'miscellaneous.title' )
+    };
 
     return this;
   },
 
-  render: function(){
+  render: function(topic){
     var section = app.views.Base.prototype.render.apply(this, arguments);
 
     // After render actions
     this.resetMenu(true);
     this.renderStaticSection("getting_help", "faq_getting_help", this.GETTING_HELP_SUBS);
 
+    var elTarget = this.findSection(topic);
+    if(elTarget != null){ $(elTarget).click(); }
```

Comment:
You see if there's already a test file in `spec/javascripts` for this view or create a new one if not.

See https://wiki.diasporafoundation.org/Testing_workflow#Jasmine